### ISSUE TITLE
Ready for testing: Downloads

### ISSUE THREAD

**Issue by eah13** [astro-pi/beta-testing#23]
To make it easier to download and run code on a Pi we've released a new Downloads improvement.

From any trinket, select the Download option in the left hand menu:
![screenshot 2016-07-29 at 12 08 20 pm](https://cloud.githubusercontent.com/assets/1702745/17255014/4b36d84c-5585-11e6-9511-77426c8ebd83.png)

This will include any files and images in the trinket in a zip file.  Additionally, if you import sense_hat it will include a folder with the latest copy of the sense_hat module in it.  If you then run the program _from the unzipped downloads directory_\* it will use this instead of your locally installed sense_hat, obviating the need to `pip install` sense_hat 2.2, the version with the new stick API.

In other words: you shouldn't need to know or care what version of sense_hat your Pi has.

There are limitations to this approach, and we're still working on support for Epiphany and Midori.  For now, make sure you've `sudo apt-get install iceweasel`ed and test in Iceweasel.  Should work fine on all other major browsers if you want to transfer to your pi some other way.
